### DISCUSSION TITLE
Help with Snacks.input Config

### DISCUSSION THREAD

**Discussion by jblsp** [folke/snacks.nvim#284] [Q&A] [answered]
I'm trying to configure Snacks.input so it's similar to the vanilla vim.ui.input. I'm having trouble with the window position. I'd like to display the window under the statusline. Also with my current window position I cannot figure out how to display the window title. Displaying the title where the icon currently is would be ideal. I really would appreciate any help with this.
```lua
    input = { },
    styles = {
      input = {
        position = "bottom",
        height = 2,
        title_pos = "left",
        keys = {
          esc = { "<esc>", { "cmp_close", "cancel" } },
          i_esc = { "<esc>", "stopinsert", mode = "i" },
          i_cr = { "<cr>", { "cmp_accept", "confirm" }, mode = "i" },
          i_tab = { "<tab>", { "cmp_select_next", "cmp" }, mode = "i" },
        },
      },
    },
```

**Comment by folke** [comment, marked as answer]
Just pushed some updates. You can achieve what you want with:

```lua
      input = {
        prompt_pos = "left",
        icon_pos = "left",
        expand = false,
      },
      styles = {
        input = {
          border = "none",
          row = -1,
          width = 0,
          wo = {
            winhighlight = "NormalFloat:StatusLine",
          },
        },
      },
```
Best not to use `position="bottom"`, since that would open a split.

**Reply by jblsp** [reply]
This is really cool. Thank you.

**Comment by jude7733** [comment]
_checkhealth_
- ERROR `vim.ui.input` is not set to `Snacks.input`

How to fix this? I tried setting to 

`vim.ui.input = Snacks.input
`
